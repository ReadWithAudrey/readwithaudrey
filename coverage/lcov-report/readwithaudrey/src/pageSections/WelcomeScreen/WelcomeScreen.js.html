<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for readwithaudrey/src/pageSections/WelcomeScreen/WelcomeScreen.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../../prettify.css" />
    <link rel="stylesheet" href="../../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../../index.html">All files</a> / <a href="index.html">readwithaudrey/src/pageSections/WelcomeScreen</a> WelcomeScreen.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>3/3</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>1/1</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>3/3</span>
      </div>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React from 'react'
import Welcome from './WelcomeScreen.style'
import { Link } from 'gatsby'
import { Button } from '../../components/'
import PropTypes from 'prop-types'
import bookpeople from '../../images/testrwalogo.png'
&nbsp;
const WelcomeScreen = () =&gt; (
  &lt;Welcome&gt;
    &lt;div
      id="homepage-top"
      className="flex flex-column items-center justify-around w-100"
    &gt;
      &lt;h1 className="pink tc f1 montserrat lh-title m0"&gt;
        Read Aloud
        &lt;br /&gt; Together
      &lt;/h1&gt;
      &lt;Link to="/Form1/" className="white no-underline w-100"&gt;
        &lt;Button type="register" className="mt3"&gt;
          Get started
        &lt;/Button&gt;
      &lt;/Link&gt;
    &lt;/div&gt;
    &lt;img src={bookpeople} alt="Cartoon of people reading a book" /&gt;
  &lt;/Welcome&gt;
)
&nbsp;
WelcomeScreen.propTypes = {
  siteTitle: PropTypes.string,
}
&nbsp;
export default WelcomeScreen
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Fri Oct 12 2018 12:51:26 GMT+0100 (British Summer Time)
</div>
</div>
<script src="../../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../../sorter.js"></script>
</body>
</html>
