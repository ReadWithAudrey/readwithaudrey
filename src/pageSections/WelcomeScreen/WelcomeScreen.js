import React from 'react'
import Welcome from './WelcomeScreen.style'
import { Link } from 'gatsby'
import { Button } from '../../components/'
import PropTypes from 'prop-types'
import bookpeople from '../../images/testrwalogo.png'

const WelcomeScreen = () => (
  <Welcome>
    <div
      id="homepage-top"
      className="flex flex-column items-center justify-around-ns justify-start w-100 absolute-ns top-1 vh-100"
    >
      <h1 className="pink tc f1 montserrat lh-title pt4 pt3-ns f-subheadline-ns">
        Read Aloud
        <br /> Together
      </h1>
      <h2 className="green tc f4 montserrat m0 f3-ns">
        Let&apos;s find you a reading partner
      </h2>
      <Link to="/Form1/" className="white no-underline w-100">
        <Button style="register" className="mt3">
          Get started
        </Button>
      </Link>
    </div>
    <img src={bookpeople} className="mt7 w-auto img-no-margin" alt="Cartoon of people reading a book" />
  </Welcome>
)

WelcomeScreen.propTypes = {
  siteTitle: PropTypes.string,
}

export default WelcomeScreen
