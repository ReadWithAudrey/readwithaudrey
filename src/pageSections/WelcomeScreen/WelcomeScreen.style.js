import styled from 'styled-components'

const Welcome = styled.div.attrs({
  className:
    'bg-white flex flex-column items-center justify-around w-100 vh-100 c-padding overflow',
})``
export default Welcome
