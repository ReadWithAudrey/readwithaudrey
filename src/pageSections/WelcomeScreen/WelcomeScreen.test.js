import React from 'react'
import renderer from 'react-test-renderer'
import WelcomeScreen from './WelcomeScreen'

describe('Homepage component snapshot test', () =>
  it('renders correctly', () => {
    const tree = renderer.create(<WelcomeScreen />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
