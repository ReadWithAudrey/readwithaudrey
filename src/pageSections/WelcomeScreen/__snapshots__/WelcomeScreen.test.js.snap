// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage component snapshot test renders correctly 1`] = `
<div
  className="sc-bdVaJa bg-white flex flex-column items-center justify-around w-100 vh-100 c-padding bDWFJH"
>
  <div
    className="flex flex-column items-center justify-around w-100"
    id="homepage-top"
  >
    <h1
      className="pink tc f1 montserrat lh-title m0"
    >
      Read Aloud
      <br />
       Together
    </h1>
    <h2
      className="green tc f4 montserrat m0"
    >
      Let's find you a reading partner
    </h2>
    <Link
      className="white no-underline w-100"
      to="/Form1/"
    >
      <button
        className="sc-dnqmqq bg-pink hover-bg-orange white pa2 br2 w-50 h3 f3 mt4 center kQARHP"
      >
        Get started
      </button>
    </Link>
  </div>
  <img
    alt="Cartoon of people reading a book"
    src="test-file-stub"
  />
</div>
`;
